lib/run-status.js

```diff
@@ -126,6 +128,17 @@ RunStatus.prototype.handleTest = function (test) {
 	this.emit('test', test, this);
 };
 
+RunStatus.prototype.formatAssertionError = function (error) {
```

Comment:
since other improvements will be added in a second moment (like a diff view for larger objects / strings), maybe better option is a individual reporter.